dockerfiles/bridges-cumulus/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM docker.io/library/alpine:3.17.2
+
+ARG VCS_REF=master
+ARG BUILD_DATE=""
+ARG REGISTRY_PATH=docker.io/paritytech
+
+# metadata
+LABEL io.parity.image.authors="devops-team@parity.io" \
+	io.parity.image.vendor="Parity Technologies" \
+	io.parity.image.title="${REGISTRY_PATH}/bridges-cumulus" \
+	io.parity.image.description="Image to use for Bridges Cumulus scripts;" \
+	io.parity.image.source="https://github.com/paritytech/scripts/blob/${VCS_REF}/\
+dockerfiles/bridges-cumulus/Dockerfile" \
+	io.parity.image.documentation="https://github.com/paritytech/scripts/blob/${VCS_REF}/\
+dockerfiles/bridges-cumulus/README.md" \
+	io.parity.image.revision="${VCS_REF}" \
+	io.parity.image.created="${BUILD_DATE}"
+
+RUN apk add --no-cache bash yarn npm jq wget && \
+    yarn global add @polkadot/api-cli
```

Comment:
The image looks very useful, not only for bridge, maybe we can call it `polkadotjs-cli`?

Reply:
That's a good idea! I had a hard time thinking for a proper name for it